.github/workflows/publish.yml

```
@@ -326,24 +326,24 @@ jobs:
+          # monochange's discovery cannot reconcile the member, so publish it
+          # directly with the same crates_io OIDC token. Skip when the exact
+          # version already exists on crates_io so reruns stay idempotent.
+          if curl -fsSL -A "pina-release-publish" \
+            "https://index.crates.io/p/in/a/pina_test" \
+            | grep -q '"vers":"0.12.0"'; then
+            echo "pina_test 0.12.0 already published; skipping"
+            exit 0
+          fi
+          cargo publish --locked --manifest-path crates/pina_test/Cargo.toml
         shell: devenv shell -- bash -e {0}
+      - name: publish packages with monochange
```

actionlint (1.7.12)

[error] step must run script with "run" section or run action with "uses" section

(syntax-check)